torch/_inductor/codegen/wrapper.py

```diff
@@ -774,7 +775,22 @@ def generate_kernel_call(
                 stream_ptr = f"c_void_p({stream_name})"
                 self.writeline(f"{name}.{name}({call_args_str}, {stream_ptr})")
         else:
-            self.writeline(self.wrap_kernel_call(name, call_args))
+            if V.graph.aot_mode and config.aot_inductor.abi_compatible:
```

Comment:
I usually try to avoid blending `V.graph.aot_mode` specific logic into the base WrapperCodeGen class, so maybe you can create an overridden `generate_kernel_call` in CppWrapperCodeGen.